app/tests/test_config.py

```diff
@@ -136,6 +172,10 @@ def test_aws_secrets_manager_config_initialized(monkeypatch):
             "DB_NAME": "test_db_name",
             "DB_SSL_ROOT_CERTIFICATE": "test_db_ssl_root_certificate",
             "DEFAULT_PAGE_SIZE": "test_default_page_size",
+            "OPEN_SEARCH_MASTER_ROLE_ARN": "test_master_role_arn",
+            "OPEN_SEARCH_HOST": "test_os_host",  # pragma: allowlist secret
+            "OPEN_SEARCH_USERNAME": "test_os_username",
```

Comment:
username and password dont need to be set in the secret as we build the AWS4Auth object for aws